docs/specs/build/xref.yml

```diff
@@ -0,0 +1,11 @@
+---
+# Unknown xrefs are output as is with a warning
+inputs:
+  docfx.yml:
+  docs/a.md: Link to @xref
+outputs:
+  docs/a.json: |
+    { "content": "<p>Link to @xref</p>" }
+  build.log: |
+    ["warning","xref-not-found","Cannot find uid 'xref' using xref '@xref'","docs/a.md"]
```

Comment:
@xref needn't warning as it's prone to false positive

Reply:
Does <xref> syntax show warnings?